src/shared/components/oncoprint/controls/OncoprintControls.tsx

```diff
@@ -471,6 +473,11 @@ export default class OncoprintControls extends React.Component<
                 this.props.handlers.onClickClusterHeatmap &&
                     this.props.handlers.onClickClusterHeatmap();
                 break;
+            case EVENT_KEY.viewNGCHM:
+            	if (this.props.state.ngchmButtonActive && this.props.handlers.onClickNGCHM) {
+                    this.confirmModal!.show(this.props.handlers.onClickNGCHM);
```

Comment:
this is a bit of a violation of state management practice.  The state of this modal should really be kept by parent component and passed in as a prop (or dealt with entirely outside the modal.

Reply:
Passed as prop.  See what you think now.  (Still in RC)